content/applications/finance/accounting/taxes.rst

```diff
@@ -307,6 +307,32 @@ directly to a product line.
    .. image:: taxes/list-sequence.png
       :alt: The taxes' sequence in Odoo determines which tax is applied first
 
+Extra taxes
+===========
+
+"Extra taxes" is a broad term referring to additional taxes beyond the standard or basic taxes
+imposed by governments. These extra taxes can be **luxury** taxes, **environmental** taxes,
+**import** or **export duties** taxes, etc.
+
+.. note::
+   The method to compute these taxes varies across different countries. We recommend consulting your
+   country's regulations to understand how to calculate them for your business.
+
+To compute an extra tax in Odoo, :ref:`create a tax <taxes/configuration>`, enter a tax name, select
+a :ref:`Tax Computation <taxes/configuration>`, set an :guilabel:`Amount`, and in the
+:guilabel:`Advanced Options` tab, check :guilabel:`Affect Base of Subsequent Taxes`. Once created,
+order the tax according to the computation order of your country.
```

Comment:
This last sentence is super important but not clear enough, in my opinion.

It's worth explaining that the sequence order of the taxes as they are listed in the taxes list view is what is going to define the computation order. The example below then reinforces super well the info.